templates/forms/fields/hidden/hidden.html.twig

```diff
@@ -1,8 +1,11 @@
 {% extends "forms/field.html.twig" %}
 
+{# NOTE: evaluate is used here instead of evaluate_twig for backwards compatibility #}
+{% set default = default ?? (field.evaluate and field.default and field.default is not iterable ? evaluate(field.default) : field.default) %}
+
 {% block field %}
 
-{% set value = value ?: (field.value ?? (field.evaluate ? evaluate(field.default) : field.default)) %}
```

Comment:
@rhukster Above old code is broken (and can only be fixed in base field), which means that we kind of need to implement this one.